.github/workflows/ci.yml

```diff
@@ -1,93 +1,95 @@
 name: CI
 
 on:
+  schedule:
+    - cron: "00 19 * * *" # run ci periodically at 3 am
   pull_request:
     branches: [master]
 
 env:
   CI_RUST_TOOLCHAIN: 1.61.0
 
 jobs:
-  outdated:
-    name: Outdated
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/install@v0.1
-        with:
-          crate: cargo-outdated
-          version: latest
-      - run: cargo outdated
+   outdated:
+     name: Outdated
```

Comment:
remove the white space if it's not necessary. It's a meaningless change.

Reply:
Fixed.